playbooks/roles/rabbitmq/templates/rabbitmq-config-newrelic-infra.yml.j2

```diff
@@ -0,0 +1,21 @@
+### WARNING: ###
+# This file is managed via Ansible.  Manual changes to this file will be overwritten.
+################
+integration_name: com.newrelic.rabbitmq
+
+instances:
+  - name: "{{ RABBITMQ_HOSTNAME }}"
+    command: all
+    arguments:
+      hostname: "{{ RABBITMQ_HOSTNAME }}"
+      port: 15671
```

Comment:
Please use the [`RABBITMQ_HTTPS_PORT`](https://github.com/open-craft/ansible-playbooks/blob/e3af5face67ea9071f1de5888c0aa2f7a10e7310/playbooks/roles/rabbitmq/defaults/main.yml#L4-L5) variable instead.